olmv1.md

```diff
@@ -0,0 +1,298 @@
+
+<br/>
+<br/>
+
+---
+
+## `olmv1 create`
+Create `olmv1` resources, currently supports only `ClusterCatalogs`.
+
+```bash
+Create a resource
+
+Usage:
+  operator olmv1 create [command]
+
+Available Commands:
+  catalog     Create a new catalog
+```
+<br/>
+
+### `olmv1 create catalog`
+Create a new `ClusterCatalog` with the provided name and catalog image reference.
+
+```bash
+Create a new catalog
+
+Usage:
+  operator olmv1 create catalog <catalog_name> <image_source_ref> [flags]
+
+Aliases:
+  catalog, catalogs <catalog_name> <image_source_ref>
+
+Flags:
+      --available                          true means that the catalog should be active and serving data (default true)
+      --cleanup-timeout duration           the amount of time to wait before cancelling cleanup after a failed creation attempt (default 1m0s)
+      --labels stringToString              labels that will be added to the catalog (default [])
+      --priority int32                     priority determines the likelihood of a catalog being selected in conflict scenarios
+      --source-poll-interval-minutes int   catalog source polling interval [in minutes] (default 10)
+```
+
+The flags allow for setting most mutable fields:
+- `--available`: Sets whether the `ClusterCatalog` should be actively serving and make its contents available on cluster. Default: true.
+- `--cleanup-timeout`: If a `ClusterCatalog` creation attempt fails due to the resource never becoming healthy, `olmv1` cleans up by deleting the failed resource, with a timeout specified by `--cleanup-timeout`. Default: 1 minute (1m)
+- `--labels`: Additional labels to add to the newly created `ClusterCatalog` as `key=value` pairs. This flag may be specified multiple times.
+- `--priority`: Integer priority used for ordering `ClusterCatalogs` in case two extension packages have the same name across catalogs, with a higher value indicating greater relative priority. Default: 0
+- `--source-poll-interval-minutes`: The polling interval to check for changes if the `ClusterCatalog` source image provided is not a digest based image, i.e, if it is referenced by tag. Set to 0 to disable polling. Default: 10
+<br/>
+<br/>
+---
+## olmv1 install
+Install resources with olmv1, currently supports `ClusterExtensions`.
+
+```bash
+Install a resource
+
+Usage:
+  operator olmv1 install [command]
+
+Available Commands:
+  extension   Install an extension
+```
+<br/>
+
+### olmv1 install extension
+
+Install a new `ClusterExtension` with the provided namez.
+
+```bash
+Install an extension
+
+Usage:
+  operator olmv1 install extension <extension_name> [flags]
+
+Flags:
+  -c, --channels strings           channels which would be used for getting updates, e.g, --channels "stable,dev-preview,preview"
+  -d, --cleanup-timeout duration   the amount of time to wait before cancelling cleanup after a failed creation attempt (default 1m0s)
+  -n, --namespace string           namespace to install the operator in
```

Comment:
if separate from the cli help text, would reword to something like "installation namespace for the operator" or "destination namespace for the operator installation"